src/Hopac.Websockets/Hopac.Websockets.fs

```diff
@@ -214,12 +215,15 @@ module ThreadSafeWebSocket =
         let createFromWebSocket (webSocket : WebSocket) =
 
             let self = {
+                websocket = webSocket
                 sendCh = Ch ()
                 receiveCh = Ch ()
                 closeCh = Ch ()
                 closeOutputCh = Ch ()
             }
 
+            let webSocket = self.websocket
```

Comment:
whats the need for these two lines?

Reply:
To allow access to it to an outside caller.

Reply:
How does this allow access? You're already putting it in the record type.

Reply:
I thought that you needed to do this to allow access to it. I'll fix it